src/components/Breadcrumbs/Breadcrumb.js

```diff
@@ -11,7 +11,6 @@ const Breadcrumb = ({ children, className, variant, ...props }) => (
       className={classNames(className, `breadcrumb--${variant}`)}
       itemScope
       itemType="http://schema.org/Breadcrumb"
-      childrenLen={children && Array.from(children).length}
```

Comment:
If we were to keep it, this should change to  React.Children.count()

Reply:
We do need this for truncation to work as expected.